lib/config/validation.spec.ts

```diff
@@ -145,6 +145,18 @@ describe(getName(__filename), () => {
       expect(errors).toHaveLength(2);
       expect(errors).toMatchSnapshot();
     });
+    it('warning if included not supported enabled managers', async () => {
+      const config = {
+        enabledManagers: ['foo'],
```

Comment:
```suggestion
        enabledManagers: ['foo', 'maven', 'gradle', 'nmp'],
```

Test with more than one, and also with a mix. Also add a valid `enabledManagers` to another test so that we can confirm it does not error when only valid are included